app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -54,6 +55,8 @@ class GithubWebhook extends RequestHandler<Body> {
         await config.deviceLabServiceAccount;
     final LuciBuildService luciBuildService =
         LuciBuildService(config, buildBucketClient, serviceAccountInfo);
+    final GithubStatusService githubStatusService =
```

Comment:
Consider making `githubStatusService` a member variable as `GithubWebhook` class naturally needs to interact with github in multiple member methods.

Reply:
Done